Forms/Forms.stencil

```diff
@@ -0,0 +1,50 @@
+{# One file per Form #}
+
+{% for type in types.based.Form|!protocol|annotated:"form" %}
+// sourcery:file:Forms/{{ type.localName }}/{{ type.localName }}+Form.generated.swift
+import Forms
+
+extension {{ type.localName }} {
+    {{ type.accessLevel }} var fields: [FieldType] {
+        return [
+{% for variable in type.variables where variable.typeName|hasPrefix:"FormField<" and variable|!annotated:"ignoreField" %}
```

Comment:
Could we check for type instead of typeName here? I can't remember the syntax, but maybe something like `type.based.FormField` or similar

Reply:
I think I did it this way to include the generic part of the type.

Reply:
Yeh I tried it and that doesn't work ..